core/src/packet.rs

```diff
@@ -427,11 +427,12 @@ impl Blob {
     /// A bool is used here instead of a flag because this item is not intended to be signed when
     /// blob signatures are introduced
     pub fn should_forward(&self) -> bool {
-        self.data[FORWARD_RANGE][0] & 0x1 == 1
+        self.data[FORWARDED_RANGE][0] & 0x1 == 0
```

Comment:
can you do the inverse of u8::from(bool) here?

Reply:
not afaik.

Reply:
I think, then the test is "== 1"

Reply:
If "has been forwarded already" (we just got it off the wire), then "should_forward" is false.